app/code/community/Nexcessnet/Turpentine/Model/Varnish/Configurator/Abstract.php

```diff
@@ -746,13 +746,18 @@ protected function _vcl_get_probe($probeUrl) {
             .probe = {
                 .request =
                     "GET {{probe_path}} HTTP/1.1"
-                    "Host: {{probe_host}}"
+                    "Host: {{probe_host}}"{{probe_https}}
```

Comment:
i didn't change anything on {{probe_host}}. this contains only the host part of the url, e.g. www.website.com. For clarification, i add two sample results, one with and one without https configured for the probe:

Probe url set to `http://www.website.com/test.php` results in:
```
            .probe = {
                .request =
                    "GET /test.php HTTP/1.1"
                    "Host: www.website.com"
                    "Connection: close";
            }
```

Probe url set to `https://www.website.com/test.php` results in:
```
            .probe = {
                .request =
                    "GET /test.php HTTP/1.1"
                    "Host: www.website.com"
                    "X-Forwarded-Proto: https"
                    "Connection: close";
            }
```